knative-operator/deploy/resources/knativekafka/README.md

```diff
@@ -0,0 +1,15 @@
+- Download the files from Github releases (no auto download at the moment)
```

Comment:
Simple instructions to download the KafkaChannel and KafkaSource manifests. Not really relevant to this PR, but let's merge this too.

Reply:
Not sure I get the value here, tbh

Reply:
Well, there's a `sed` command to delete comments and blank lines from manifests.
That's all :)

Reply:
Seems odd to dump on this PR but :man_shrugging: